delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchViewJsonLdSourceDecoder.scala

```diff
@@ -95,23 +94,21 @@ object ElasticSearchViewJsonLdSourceDecoder {
   }
 
   import ElasticSearchViewFields._
-  def toValue(fields: ElasticSearchViewFields): UIO[ElasticSearchViewValue] = fields match {
+  private def toValue(fields: ElasticSearchViewFields): ElasticSearchViewValue = fields match {
     case i: IndexingElasticSearchViewFields  =>
-      defaultElasticsearchSettings.map { defaultSettings =>
-        IndexingElasticSearchViewValue(
-          resourceSchemas = i.resourceSchemas,
-          resourceTypes = i.resourceTypes,
-          resourceTag = i.resourceTag,
-          sourceAsText = i.sourceAsText,
-          includeMetadata = i.includeMetadata,
-          includeDeprecated = i.includeDeprecated,
-          mapping = i.mapping,
-          settings = i.settings.getOrElse(defaultSettings),
-          permission = i.permission
-        )
-      }
+      IndexingElasticSearchViewValue(
+        resourceSchemas = i.resourceSchemas,
+        resourceTypes = i.resourceTypes,
+        resourceTag = i.resourceTag,
+        sourceAsText = i.sourceAsText,
+        includeMetadata = i.includeMetadata,
+        includeDeprecated = i.includeDeprecated,
+        mapping = Some(i.mapping),
```

Comment:
Tis means that the `mapping` field for the API client it is still mandatory right? I guess it is ok.

Reply:
Yep, it is still mandatory